tests/blockchain/eth/contracts/main/mining_adjudicator/test_mining_adjudicator.py

```diff
@@ -92,28 +94,42 @@ def fragments(metadata):
     return capsule, cfrag
 
 
+def compute_penalty_and_reward(stake: int, penalty_history: int) -> Tuple[int, int]:
+    penalty = BASE_PENALTY + PENALTY_HISTORY_COEFFICIENT * penalty_history
+    penalty = min(penalty, stake // PERCENTAGE_PENALTY_COEFFICIENT)
+    reward = penalty // REWARD_COEFFICIENT
+    return penalty, reward
+
+
 @pytest.mark.slow
 def test_evaluate_cfrag(testerchain, escrow, adjudicator_contract):
     creator, miner, wrong_miner, investigator, *everyone_else = testerchain.interface.w3.eth.accounts
     evaluation_log = adjudicator_contract.events.CFragEvaluated.createFilter(fromBlock='latest')
 
+    # TODO: Uncomment assertions related to old public methods, now internal
+
     # TODO: Move this to an integration test?
     umbral_params = default_params()
     u_xcoord, u_ycoord = umbral_params.u.to_affine()
     u_sign = 2 + (u_ycoord % 2)
-    assert u_sign == adjudicator_contract.functions.UMBRAL_PARAMETER_U_SIGN().call()
-    assert u_xcoord == adjudicator_contract.functions.UMBRAL_PARAMETER_U_XCOORD().call()
-    assert u_ycoord == adjudicator_contract.functions.UMBRAL_PARAMETER_U_YCOORD().call()
+    # assert u_sign == adjudicator_contract.functions.UMBRAL_PARAMETER_U_SIGN().call()
+    # assert u_xcoord == adjudicator_contract.functions.UMBRAL_PARAMETER_U_XCOORD().call()
+    # assert u_ycoord == adjudicator_contract.functions.UMBRAL_PARAMETER_U_YCOORD().call()
```

Comment:
Since this library is separate it makes sense to make test-wrapper for library and transfer some of the tests to lib folder

Reply:
Agreed. I'll make the changes.